.github/.syncignore

```diff
@@ -0,0 +1,3 @@
+CODEOWNERS
+dependabot.yml
+workflows/push-image.yml
```

Comment:
```suggestion
```
There's now a parameterized push builder image workflow that will work for this repo:
https://github.com/paketo-buildpacks/github-config/commit/3b0bfa0fbf3366f54d7e1a0952a310c0371b308b

See the workflow in action here: https://github.com/paketo-buildpacks/builder-jammy-buildpackless-tiny/actions/runs/2666153033

Reply:
Ok let me fix that up

Reply:
Updated!